db/queries.py

```diff
@@ -1,23 +1,40 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from django.db.models import F, Q
+from db.models import LoyaltyProgram, Customer, LoyaltyProgramParticipant
 
 
 def all_loyalty_program_names() -> QuerySet:
-    pass
+    return LoyaltyProgram.objects.all().values_list("name", "bonus_percentage")
```

Comment:
```suggestion
    return LoyaltyProgram.objects.values_list("name", "bonus_percentage")
```
extra `all` call

fix everywhere